backend/YouTubeV2/Program.cs

```diff
@@ -13,7 +14,7 @@
 
 // Add services to the container.
 
-builder.Services.AddControllers();
+builder.Services.AddControllers(options =>  options.Filters.Add(new ProducesAttribute("application/json")));
```

Comment:
what does this do?

Reply:
It shows the response type as "application/json" in swagger

Reply:
But do we always want to have application/json. I think not